source/_components/neato.markdown

```diff
@@ -64,6 +64,7 @@ Currently supported services are:
 - `return_to_base`
 - `locate`
 - `spot_clean`
+- `neato_custom_cleaning` (The `zone` parameter is only supported for the Botvac D7)
```

Comment:
I wonder if we should provide a table of what all the values mean and which values are supported by which botvac models.  Maybe for the time being we just link to this page so users can see a better description?  The only issue is that actual botvac models are not listed here.

https://developers.neatorobotics.com/api/robot-remote-protocol/housecleaning

Reply:
Done in the latest commit